app/api/controllers/RequestHandler.scala

```diff
@@ -109,19 +109,31 @@ object RequestHandler {
           message = s"[${ctx.endpointLogContext.controllerName}][${ctx.endpointLogContext.endpointName}] " +
             s"with correlationId : ${ctx.correlationId}")
 
-        val result =
-          for {
-            parsedRequest   <- EitherT.fromEither[Future](parser.parseRequest(rawData))
-            serviceResponse <- EitherT(service(parsedRequest))
-          } yield doWithContext(ctx.withCorrelationId(serviceResponse.correlationId)) { implicit ctx: RequestContext =>
-            handleSuccess(rawData, parsedRequest, serviceResponse)
-          }
-
-        result.leftMap { errorWrapper =>
-          doWithContext(ctx.withCorrelationId(errorWrapper.correlationId)) { implicit ctx: RequestContext =>
-            handleFailure(errorWrapper)
-          }
-        }.merge
+        val maybeGovTestScenario = ctx.hc.otherHeaders.find(header => header._1 == "Gov-Test-Scenario") match {
+          case Some(heading) => Some(heading._2)
+          case None => None
+        }
+
+        if(maybeGovTestScenario.contains("REQUEST_CANNOT_BE_FULFILLED")){
```

Comment:
Can the code be formatted using scalafmt?  It's worth setting it up to "format on save" in Intellij, + make sure it's using scalafmt rather than intellij's own formatter